Core/src/main/java/com/github/intellectualsites/plotsquared/plot/generator/HybridPlotManager.java

```diff
@@ -103,6 +104,9 @@ private void createSchemAbs(HybridPlotWorld hpw, LocalBlockQueue queue, Location
                     for (int y = 0; y < blocks.length; y++) {
                         if (blocks[y] != null) {
                             queue.setBlock(x, minY + y, z, blocks[y]);
+                        } else {
+                            // This is necessary, otherwise any blocks not specified in the schematic will remain after a clear
+                            queue.setBlock(x, minY + y, z, BlockTypes.AIR.getDefaultState().toBaseBlock());
```

Comment:
This air block should probably be extracted out into a local variable outside the loop for the sake of performance, right?

Reply:
I would think the java compiler would do this for us but I'm not going to pull this change until late Friday. Go ahead and make it a variable in another commit if you want.

Reply:
Alright, so I looked into the WE code, and it wouldn't be all that inefficient, since there's caching in there, but I still think Java has to make the method calls each time, since I'd be amazed if it could somehow deduce that there aren't side effects in them, so this should in theory save some method calls at least.